Sources/InjectAutoLoader/InjectAutoLoader.m

```diff
@@ -0,0 +1,17 @@
+#import <Foundation/Foundation.h>
+
+@import Inject;
+
+@interface InjectAutoLoader: NSObject
++ (void)load;
+@end
+
+@implementation InjectAutoLoader
++ (void)load
+{
+#ifdef DEBUG
```

Comment:
Can we put the #if DEBUG around everything in this file including the @import?

Reply:
I pushed updates. 

However, I'm not sure about the second part. I'm not very fluent with Objective-C, but I think it doesn't make a big difference, does it?